R/rd2roxygen.R

```diff
@@ -36,7 +36,11 @@ parse_file = function(path) {
   out$seealso = reconstruct(untag(rd$seealso))
   out$references = reconstruct(untag(rd$references))
   out$source = reconstruct(untag(rd$source))
-
+  out$concepts <- unname(do.call(c, lapply(rd[names(rd) == "concept"], function(x) {
```

Comment:
Is `do.call(c)` equivalent to `unlist()` here? Normally I avoid `do.call()` when possible.